src/utils/attachments.ts

```diff
@@ -0,0 +1,39 @@
+import { flatten } from "fp-ts/lib/Array";
+import { CreatedMessageWithContentAndAttachments } from "generated/backend/CreatedMessageWithContentAndAttachments";
+import { PrescriptionData } from "generated/backend/PrescriptionData";
+import { CreatedMessageWithContent } from "generated/io-api/CreatedMessageWithContent";
+import { ResponseSuccessJson } from "italia-ts-commons/lib/responses";
+import { toBarcode } from "./barcode";
+
+const toMessageAttachments = async (name: string, value: string) => {
+  const errorOrBarcodes = await toBarcode(value);
+  return errorOrBarcodes.fold(
+    () => [],
+    barcodes => [
+      { name, content: barcodes.png, mime_type: barcodes.pngMimeType },
+      { name, content: barcodes.svg, mime_type: barcodes.svgMimeType }
+    ]
+  );
+};
+
+export const fillMessageAttachmentsPayload = async (
```

Comment:
ditto

Reply:
moreover this method name does not reflect its implementation: it not only creates the array with the attachments but also wrap its result into a responsesuccessjson (which is not its responsibility; the controller must do that, here it's fair to just return the object)